### DISCUSSION TITLE
Support for math-blocks with Katex/MathJax

### DISCUSSION THREAD

**Discussion by Treeniks** [retypeapp/retype#72] [Ideas]
Hi, I think Retype would be well suited to create websites about mathematical topics, but for that to be feasible it would need proper support for some kind of HTML Math rendering, of which I couldn't find any documentation or examples for (so I assume it's not supported as of yet). The two most popular in that area would be Katex and MathJax (although I personally much prefer Katex). Would it be possible to include support for these?

Being able to do something like `$s_n = \sum_{k = 0}^{n} a_k$` inside the markdown source and get a nice Katex-rendered math formula in the website would be a great addition I believe.

**Comment by geoffreymcgill** [comment]
Logged this as a new feature request issue #73.

**Reply by geoffreymcgill** [reply]
Full KaTex support has been added and will be available within the upcoming Retype `v1.10.0` release.
